### PR TITLE
[13.0][MIG] account_balance_line

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

**Review by schout-it — APPROVED** [review]
lgtm

File: account_balance_line/README.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line
+    :alt: OCA/account-financial-tools
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_balance_line
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/92/12.0
```
Comment (CasVissers) [inline account_balance_line/README.rst]: 13.0?

File: account_balance_line/README.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line
+    :alt: OCA/account-financial-tools
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_balance_line
```
Comment (CasVissers) [inline account_balance_line/README.rst]: 13.0?

File: account_balance_line/README.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/account-financial-tools <https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line>`_ project on GitHub.
```
Comment (CasVissers) [inline account_balance_line/README.rst]: 13.0?

File: account_balance_line/i18n/account_balance_line.pot [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* account_balance_line
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (CasVissers-360ERP) [inline account_balance_line/i18n/account_balance_line.pot:7]: Is this pot correct?

File: account_balance_line/README.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line
```
Comment (CasVissers-360ERP) [inline account_balance_line/README.rst:17]: 13.0?

File: account_balance_line/README.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+#. Or open an account or a partner, and click on Action > Journal items.
+#. You will see a column named "Balance" with the balance of the line.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-financial-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/account-financial-tools/issues/new?body=module:%20account_balance_line%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (CasVissers) [inline account_balance_line/README.rst]: 13.0?

File: account_balance_line/static/description/index.html [hunk 7]
```diff
@@ -0,0 +1,433 @@
...
+!! changes will be overwritten.                   !!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Beta" src="https://img.shields.io/badge/maturity-Beta-yellow.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/agpl-3.0-standalone.html"><img alt="License: AGPL-3" src="https://img.shields.io/badge/licence-AGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line"><img alt="OCA/account-financial-tools" src="https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_balance_line"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/92/12.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
```
Comment (CasVissers-360ERP) [inline account_balance_line/static/description/index.html:370]: 13.0?

File: account_balance_line/static/description/index.html [hunk 8]
```diff
@@ -0,0 +1,433 @@
...
+</ul>
+</div>
+<div class="section" id="maintainers">
+<h2><a class="toc-backref" href="#id6">Maintainers</a></h2>
+<p>This module is maintained by the OCA.</p>
+<a class="reference external image-reference" href="https://odoo-community.org"><img alt="Odoo Community Association" src="https://odoo-community.org/logo.png" /></a>
+<p>OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.</p>
+<p>This module is part of the <a class="reference external" href="https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line">OCA/account-financial-tools</a> project on GitHub.</p>
```
Comment (CasVissers-360ERP) [inline account_balance_line/static/description/index.html:427]: 13.0?

**Review by CasVissers-360ERP — CHANGES_REQUESTED** [review]
Don't know if necessary but everthing in the module still points to 12.0?

File: account_balance_line/README.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [inline account_balance_line/README.rst]: not necessary because the readme will be nicely auto-updated by our ocabot after merge 😎

**Comment by kittiu** [comment]
Hi, I am not quite sure why the Travis is not green.
But anyway, it seem `pre-commit` is not used yet, can you fix that?
It will rerun the travis too. :)

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM!

**Comment by kittiu** [comment]
@feketemihai 
I think, at least, this would need to rebase to latest 13.0
pre-commit is not working with your branch.

**Comment by feketemihai** [comment]
@kittiu Done, rebase against 13.0

**Pull request merged**